doc/custom_conf.py

```diff
@@ -182,11 +182,13 @@
     'https://www.dell.com/en-us/shop/powerflex/sf/powerflex',
     'https://www.gnu.org/licenses/agpl-3.0.en.html',
     r"https://ceph\.io(/.*)?",
-    # 403 from GH runners
+    # Blocked from GH runners
     'https://www.schlachter.tech/solutions/pongo2-template-engine/',
-    # Cloudflare protection on SourceForge domains might block linkcheck
     r"https://.*\.sourceforge\.net/.*",
     'https://www.hpe.com/emea_europe/en/hpe-alletra.html'
+    r'https://www.canonical.com.*',
```

Comment:
Many of the redirection I see don't have the `www.` prefix so here's a more generic regex:
```suggestion
    r'https://(www\.)?canonical\.com/.*',
```